website/docs/patterns/table-multi-select/partials/accessibility/accessibility.md

```diff
@@ -0,0 +1,33 @@
+!!! Info
+
+It is the responsibility of consumers to implement this pattern, and therefore also their responsibility to ensure that all accessibility success criteria are met or exceeded.
+!!!
+
+## Aria role
+
+Due to its dynamic nature, updates to the selected count must be announced to those using assistive technology. This can be done by adding [`role=status`](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA22) to the element containing the count.
+
+This is a _representative_ example of how this could be accomplished using HDS components and `role=status` to announce changes in the UI. Test it yourself by enabling voice over and incrementing the `selectedCount` with the "Add to count" button.
+
+```handlebars
+<div class="doc-table-multi-select-role-example">
+
```

Comment:
```suggestion
```

Reply:
Super nit: This is just my personal opinion but if you indent a block of code then blank lines above and below shouldn't be necessary. (It should be either/or)